docs/reference.md

```diff
@@ -966,6 +983,196 @@ users from seeing authenticated pages momentarily during page load. See the
 ["Using Authentication with Routers"](/docs/guide/user-auth.md#authenticating-with-routers)
 section of our AngularFire guide for more information and a full example.
 
+## $firebaseStorage
+
+AngularFire includes support for [binary storage](/docs/guide/uploading-downloading-binary-content.md)
+with the `$firebaseStorage` service.
+
+The `$firebaseStorage` service takes a [Firebase Storage](https://firebase.google.com/docs/storage/) reference.
+
+```js
+app.controller("MyCtrl", ["$scope", "$firebaseStorage",
+  function($scope, $firebaseStorage) {
+    var storageRef = firebase.storage().ref("images/dog");
+    $scope.storage = $firebaseStorage(storageRef);
+  }
+]);
+```
+
+The storage object returned by `$firebaseStorage` contains several methods for uploading and
+downloading binary content, as well as managing the content's metadata.
+
+### $put(file, metadata)
+
+[Uploads a `Blob` object](https://firebase.google.com/docs/storage/web/upload-files) to the specified storage reference's path with an optional metadata parameter.
+Returns an [`UploadTask`](#upload-task) wrapped by AngularFire.
+
+
+```js
+var htmlFile = new Blob(["<html></html>"], {type : "text/html"});
```

Comment:
Inconsistent use of spacing: `{foo:"bar"}` in this line but `{ foo: "bar" }` in the next line. In this guide, we typically have the space padding.